docker-compose.yml

```diff
@@ -0,0 +1,81 @@
+version: '3'
+
+services:
+
+  elasticsearch1:
+    image: docker.elastic.co/elasticsearch/elasticsearch:5.3.1
+    container_name: elasticsearch1
+    ports:
+      - 9200:9200
+      - 9300:9300
```

Comment:
These ports are being exposed without being configurable - this breaks the ease of deployment mandate.